docs/source/app_developers_guide/go_sdk_import.rst

```diff
@@ -0,0 +1,59 @@
+********************
+Importing the Go SDK
+********************
+
+.. note::
+   The Sawtooth Go SDK assumes that you have the latest version of Go,
```

Comment:
The punctuation is incorrect. A simple fix would be to replace the comma with a period and capitalize the m in "more":

"... that you have the latest version of Go. More information can be..."